projects/packages/my-jetpack/_inc/components/onboarding-tour/index.tsx

```diff
@@ -0,0 +1,107 @@
+import useWelcomeTour from './use-welcome-tour';
+
+const OnboardingTour = () => {
+	const { dismissWelcomeTour, isWelcomeTourVisible } = useWelcomeTour();
+	const [ isOpen, setOpen ] = useState( isWelcomeTourVisible );
+
+	const closeGuide = useCallback( () => {
+		dismissWelcomeTour();
+		setOpen( false );
+	}, [ dismissWelcomeTour, setOpen ] );
+
+	return (
+		<>
+			{ isOpen && (
+				<Guide
+					className="myjetpack-onboarding-welcome-tour"
+					contentLabel={ __( 'Welcome to Jetpack', 'jetpack-my-jetpack' ) }
+					onFinish={ closeGuide }
+					finishButtonText={ __( 'Done', 'jetpack-my-jetpack' ) }
+					pages={ [
+						{
+							image: <WelcomeTourImage nonAnimatedSrc={ TourImage1 } animatedSrc={ TourImage1 } />,
+							content: (
+								<>
+									<h1 className="myjetpack-onboarding-welcome-tour__heading">
+										{ __( 'Simple, yet powerful stats', 'jetpack-my-jetpack' ) }
+									</h1>
+									<p className="myjetpack-onboarding-welcome-tour__text">
+										{ __(
+											'In a few hours you’ll see detailed insights on who’s visiting your site and where they’re coming from.',
+											'jetpack-my-jetpack'
+										) }
+									</p>
+								</>
+							),
+						},
+						{
+							image: <WelcomeTourImage nonAnimatedSrc={ TourImage2 } animatedSrc={ TourImage2 } />,
+							content: (
+								<>
+									<h1 className="myjetpack-onboarding-welcome-tour__heading">
+										{ __( 'Making your site super fast', 'jetpack-my-jetpack' ) }
+									</h1>
+									<p className="myjetpack-onboarding-welcome-tour__text">
+										{ __(
+											'We’re already checking how your site stacks up and will suggest speed optimizations in a few minutes.',
+											'jetpack-my-jetpack'
+										) }
+									</p>
+								</>
+							),
+						},
+						{
+							image: <WelcomeTourImage nonAnimatedSrc={ TourImage3 } animatedSrc={ TourImage3 } />,
+							content: (
+								<>
+									<h1 className="myjetpack-onboarding-welcome-tour__heading">
+										{ __( 'Your entire site, safe in the cloud', 'jetpack-my-jetpack' ) }
+									</h1>
+									<p className="myjetpack-onboarding-welcome-tour__text">
+										{ __(
+											'We’ve started to back up your entire site in the cloud. Restore with one click if anything goes wrong.',
+											'jetpack-my-jetpack'
+										) }
+									</p>
+								</>
+							),
+						},
+						{
+							image: <WelcomeTourImage nonAnimatedSrc={ TourImage4 } animatedSrc={ TourImage4 } />,
+							content: (
+								<>
+									<h1 className="myjetpack-onboarding-welcome-tour__heading">
+										{ __( 'Your site goes wherever you go.', 'jetpack-my-jetpack' ) }
+									</h1>
+									<p className="myjetpack-onboarding-welcome-tour__text">
+										{ createInterpolateElement(
+											__(
+												'Install the Jetpack app for iOS or Android and stay connected to your site from anywhere!<br />We sent you an email with the download link!',
```

Comment:
Minor nitpick, but the two sentences ending with exclamation marks sound overly enthusiastic to me. I'd keep it in the first sentence, since the second sentence is more informative, and change the second one to a period.